crates/bevy_math/src/primitives/dim3.rs

```diff
@@ -765,6 +765,43 @@ impl Torus {
     }
 }
 
+/// A triangular prism primitive, often representing a ramp.
+#[derive(Copy, Clone, Debug, PartialEq)]
+#[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
+pub struct Prism {
+    /// Half of the width, height and depth of the prism
+    pub half_size: Vec3,
+    /// Displacement of the apex edge along the Z-axis. -1.0 positions the apex straight above the bottom-front edge.
+    pub apex_displacement: f32,
+}
```

Comment:
Maybe It's just me but I'm not familiar with these measurements for prisms and I'm kind of struggling with this API. How do I create a prism with a right or isosceles base? Can I use this to create oblique prisms, or all they all right?

I'd really appreciate some more complete struct docs and (ideally) some examples.